app/assets/stylesheets/application.css

```diff
@@ -4,12 +4,6 @@
  *
  * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
  * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
  *= require_tree .
  *= require_self
  */
```

Comment:
- [ ] Please make sure the buttons styles in the app should match the given design.

### Your design

![image](https://github.com/Bleu-Yves-Sopoude/manago/assets/83803414/2a876c29-848a-43a7-b119-a5d30e2995cc)
![image](https://github.com/Bleu-Yves-Sopoude/manago/assets/83803414/0e82e2a1-c2b2-4531-9998-51db0df49ae6)


### Given design

![image](https://github.com/Bleu-Yves-Sopoude/manago/assets/83803414/9149cd77-0c4f-4fa7-8c93-8fddf09f3927)